code/modules/power/cable.dm

```diff
@@ -27,7 +27,7 @@ By design, d1 is the smallest direction and d2 is the highest
 	anchored =1
 	var/datum/powernet/powernet
 	name = "power cable"
-	desc = "A flexible superconducting cable for heavy-duty power transfer"
+	desc = "A flexible superconducting cable for heavy-duty power transfer."

```

Comment:
To be pedantic, shouldn't it be "A flexible, superconducting..." or "A flexible and superconducting..." ?

To top off my queries here, might superconductive be more appropriate? I'm going doolally from a lack of sleep so please correct/ignore me.